packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx

```diff
@@ -131,6 +131,22 @@ export const guideCards: GuideCardConstants[] = [
     telemetryId: 'onboarding--observability--kubernetes',
     order: 11,
   },
+  {
+    solution: 'observability',
+    icon: 'uptimeApp',
```

Comment:
```suggestion
    icon: 'videoPlayer',
```

I think Uptime would want to move away from the Uptime icon. I'm suggesting using the monitor type icon instead for this card.